app/test_main.py

```diff
@@ -2,2 +1,43 @@
+import pytest
 
-# write your code here
+
+from app import main
+
+def convert_to_human(animal_age: int,
+                     first_year: int,
+                     second_year: int,
+                     each_year: int
+) -> int:
+    if animal_age < first_year:
+        return 0
+    if animal_age < first_year + second_year:
+        return 1
+    return 2 + (animal_age - first_year - second_year) // each_year
+
+
+@pytest.mark.parametrize(
+    "first_year,second_year,each_year_cat,each_year_dog",
+    [
+        (14, 9, 4, 5),
+        (15.1, 9, 4, 5),
+        (15, 8, 4, 5),
+        (15, 9.1, 4, 5),
```

Comment:
The parameter `second_year` is expected to be an integer, but here it is a floating-point number (9.1). According to the task description, these values should be integers.